src/e2e/step-definitions/external-systems.ts

```diff
@@ -41,9 +44,23 @@ When(
     await this.page.getByLabel("Sjabloon").last().click();
     await this.page.getByRole("option", { name: "OpenZaakTest" }).click();
 
-    const autofillInputTitle = this.page.locator("#title_tekstfield");
-    await autofillInputTitle.click();
-    await autofillInputTitle.fill("Document Title Text");
+    const inputTitle = this.page.locator("#title_tekstfield");
+    addedDocumentTitle = `E2E Test - Document Title Text`;
```

Comment:
Minor comment on using `added` - this suggest is has already been done, would it not be better to:
- either; use another prefix or postfix that describes what you intend to do, maybe `supplied` or `given`
- or leave the whole prefix off; does it help reading the code?
- or combine this input data into an object with all the document input fields? ie. 
```typescript
const createDocumentInput = {
  title: "E2E Test - Document Title Text",
  description: "E2E Test - Document Description Text",
  author: "E2E Test - Document Description Text"
}
```

Reply:
Noce idea! Will fix using that object and put the better naming in as well

Reply:
I made it into:

`const documentInput = {
    title: "E2E Test - Document Title Text",
    description: "E2E Test - Document Description Text",
    author: "E2E Test - Document Author Name",
};`

Reply:
Changes made in the end:

```
const documentInput = {
    title: "E2E Test - Document Title Text",
    description: "E2E Test - Document Description Text",
    author: "E2E Test - Document Author Name",
};
```